Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/buttons/Button.module.scss

```diff
@@ -57,8 +58,8 @@
   }
 
   &.disabled {
-    background-color: colors.$background-color-btn-secondary;
-    border-color: colors.$color-btn-disabled;
+    background-color: transparent !important;
+    border-color: colors.$color-btn-disabled !important;
```

Comment:
Can it be refactored so it's not necessary to add !importnat?

Reply:
I don't think it can, since a Button could be for an examply secondary or primary, which both have background-color and border-color of their own and it could be dsiabled in the same time, this way the priority of the disabled button should be above all, so I think in this specific case it would be the safe way to approach this case, having the disabled rule with highest priority with !important, since this way we can be sure that this rule will always be applied no matter what, keeping us bug-safe in other scenarios as well